qutebrowser/utils/urlutils.py

```diff
@@ -298,14 +298,28 @@ def qurl_from_user_input(urlstr):
     else:
         ipstr = urlstr.strip()
         rest = ''
+
+    # This may error out because we first use this function to validate the
+    # options (_validate_all) before the register the config...
+    try:
+        default_scheme = config.get('network', 'scheme-order')[0]
+    except KeyError:
+        default_scheme = None
```

Comment:
Bleh, chicken-egg-problem :wink:

What about adding a `no_scheme_order` (?) argument to `fuzzy_url` which gets passed to `qurl_from_user_input`, and which the `FuzzyUrl` config type uses for validating?